client/js/components/registration.js

```diff
@@ -54,19 +54,14 @@ const renderRegistrationForm = () => {
             };
 
             if (!data.firstname || !data.surname || !data.email || !data.password || !data.username) {
-                return alert( "one of the fields is empty");
+                return alert( "One of the fields is empty");
```

Comment:
Can also use the errorBar function here like `errorBar("Please fill in all fields", "error")`. I guess think of the errorBar has some sort of message bar rather than strictly catch errors :)